jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/nodes/AwsExplorerNode.kt

```diff
@@ -40,5 +38,9 @@ abstract class AwsExplorerNode<T>(val nodeProject: Project, value: T, private va
     /**
      * Called when the node is double clicked on
      */
-    open fun onDoubleClick(model: DefaultTreeModel, selectedElement: DefaultMutableTreeNode) {}
+    open fun onDoubleClick() {}
+
+    override fun toString(): String {
+        return value.toString()
```

Comment:
Surprised ktlint didn't kick off about this - it could be a block expression.

Reply:
It probably will. More commits with come to fix the tests and do a few small things. There are regressions in behavior